scripts/get-dep-lib.sh

```diff
@@ -12,7 +12,7 @@ CURRENT_LIBSSH2="libssh2-1.8.0"
 CURRENT_XSLT="v1.1.34"
 CURRENT_SQLITE="3190200"
 CURRENT_LIBXML2="v2.9.4"
-CURRENT_LIBFTDI="1.3"
+CURRENT_LIBFTDI="abd19b721f7e9b4d514ed319ece173ebc7b1ea72"
```

Comment:
Do we _need_ a git hash here? (I.e. is there no released version that will work?)